README.md

```diff
@@ -6,7 +6,27 @@ Storybook is a great tool for developing components -- and while simulated and s
 
 ## Usage
 
-The quickest way to get started is to use the proof-cli.
+The quickest way to get started: 
```

Comment:
Can we move this to a seperate section at the end? Maybe `Contributing` and link to the `Contributin.md`? The `Usage` section was intended for outside people using the library.